### DISCUSSION TITLE
How about some instructions with examples of how to actually use this tool?

### DISCUSSION THREAD

**Discussion by Jibun-no-Kage** [TheAssassin/AppImageLauncher#685] [Q&A] [answered]
### Pre-submit checks

- [X] I checked for similar issues beforehand, but could not find any. I could not add my proposal to any existing issue.
- [X] I am going to take the time to to fill in all the required details. I know that the bug report will be dismissed otherwise.

### Feature description

How about some instructions with examples of how to actually use this tool?

### Proposed solution

???

### Alternative solutions you considered

???

### Additional context

How about some instructions with examples of how to actually use this tool?  I am trying to understand how to use this tool to install gdm-settings tool.  But I can't make heads or tails of how to use this tool never mind install it.  I git cloned the project, now what?

**Comment by Jibun-no-Kage** [comment]
I found the instructions on how to install on Debian, but on Debian 12 hit the following issue...
```
# apt install software-properties-common
# add-apt-repository ppa:appimagelauncher-team/stable
# apt update
# apt install appimagelauncher
```
On add the repository...
```
# add-apt-repository ppa:appimagelauncher-team/stable
Traceback (most recent call last):
  File "/usr/bin/add-apt-repository", line 362, in <module>
    sys.exit(0 if addaptrepo.main() else 1)
                  ^^^^^^^^^^^^^^^^^
  File "/usr/bin/add-apt-repository", line 345, in main
    shortcut = handler(source, **shortcut_params)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/softwareproperties/shortcuts.py", line 40, in shortcut_handler
    return handler(shortcut, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/softwareproperties/ppa.py", line 86, in __init__
    if self.lpppa.publish_debug_symbols:
       ^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/softwareproperties/ppa.py", line 126, in lpppa
    self._lpppa = self.lpteam.getPPAByName(name=self.ppaname)
                  ^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/softwareproperties/ppa.py", line 113, in lpteam
    self._lpteam = self.lp.people(self.teamname)
                   ^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'people'
```

**Comment by mihawk90** [comment, marked as answer]
> I found the instructions on how to install on Debian, but on Debian 12 hit the following issue...

You didn't read the [install guide](https://github.com/TheAssassin/AppImageLauncher/wiki/Install-on-Ubuntu-or-Debian) properly:

> _**On Ubuntu**_ and derivatives, you can use one of the [PPAs](https://en.wikipedia.org/wiki/Ubuntu#Package_Archives)

PPAs are Ubuntu specific and not meant for Debian.

> **_On_** other **_Debian-based systems_** or if you do not want to use the PPAs, you can **_install AppImageLauncher manually_** using one of the .deb packages provided on [this latest release page](https://github.com/TheAssassin/AppImageLauncher/releases/latest).

This is also already mentioned in the Readme when you open this Repo:
https://github.com/TheAssassin/AppImageLauncher?tab=readme-ov-file#system-wide-installation

**Reply by TheAssassin** [reply]
Exactly this. @Jibun-no-Kage does not follow the instrutions here.

**Comment by TheAssassin** [comment]
Closing as this is an actual user issue. @mihawk90 pointed out how to install the tool on Debian. Plus, @Jibun-no-Kage did not follow the template and fill in the required information.
